### PR TITLE
post slug

### PR DESCRIPTION
**Pull request by megaherz**

### CODE REVIEW

File: scripts/migrations/postgres/20211206165802_post_slug.down.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+BEGIN;
+
+DROP TRIGGER trigger_post_unique_slug ON post;
+
+ALTER TABLE post
+    DROP COLUMN slug;
+
+DROP FUNCTION IF EXISTS unique_post_slug();
+
+DROP MATERIALIZED VIEW calculated_post;
```
Comment (megaherz) [inline scripts/migrations/postgres/20211206165802_post_slug.down.sql:10]: There is no CREATE or REPLACE MATERIALIZED VIEW.

File: internal/server/handler_test.go [hunk 2]
```diff
@@ -232,6 +237,7 @@ func Test_getPost(t *testing.T) {
 		"text":"text",
 		"likesCount":1,
 		"dislikesCount":2,
+        "slug": "slug",
```
Comment (mikluke) [inline internal/server/handler_test.go:240]: tabs

File: scripts/migrations/postgres/20211206165802_post_slug.down.sql (hunk 1 above)
Comment (mikluke) [reply scripts/migrations/postgres/20211206165802_post_slug.down.sql:10]: I don't remember if you must refresh it once before using

**Review by mikluke — APPROVED** [review]

File: scripts/migrations/postgres/20211206165802_post_slug.down.sql (hunk 1 above)
Comment (megaherz) [reply scripts/migrations/postgres/20211206165802_post_slug.down.sql:10]: CREATE MATERIALIZED VIEW defines a materialized view of a query. The query is executed and used to populate the view at the time the command is issued (unless WITH NO DATA is used

**Pull request merged**